.github/workflows/cd-common.yaml

```diff
@@ -126,6 +131,7 @@ jobs:
           repository: onboardiq/gh-common-workflows
           path: ./common-workflows
           fetch-depth: 1
+          ref: 'chore/doppler-project-name'
```

Comment:
Seems like we don't need this post-merge

Reply:
Yeah I added that debug [commit](https://github.com/onboardiq/gh-common-workflows/pull/59/commits/b6ea9ba7c720125839986dd4719fd5119afda7c3) to unblock nestjs-backend from using this code. That plan was to remove the commit before merging.